exec/java-exec/src/test/java/org/apache/drill/exec/store/easy/json/parser/TestJsonParserBasics.java

```diff
@@ -324,4 +327,64 @@ public void testJsonModeObjects() {
     assertFalse(fixture.next());
     fixture.close();
   }
+
+  /**
+   * Example message parser. A real parser would provide much better
+   * error messages for badly-formed JSON or error codes.
+   */
+  private static class MessageParserFixture implements MessageParser {
+
+    @Override
+    public boolean parsePrefix(TokenIterator tokenizer) {
+      assertEquals(JsonToken.START_OBJECT, tokenizer.requireNext());
+      assertEquals(JsonToken.FIELD_NAME, tokenizer.requireNext());
+      assertEquals(JsonToken.VALUE_STRING, tokenizer.requireNext());
+      if (!tokenizer.stringValue().equals("ok")) {
```

Comment:
Usually is better to start with constant to avoid NPE (I know it's just a test, just noticed).
```suggestion
      if (!"ok".equals(tokenizer.stringValue()) {
```